### DISCUSSION TITLE
Is there a place where we can see release date?

### DISCUSSION THREAD

**Discussion by jorgellose** [GoogleChrome/lighthouse#12604] [Q&A]
Today PageSpeed started using lighthouse 8.0 and we experimented a decrease of 10 points (now lighthouse detects javascript downloaded 2s after all is fully loaded, so now google analytics and google adwords, both non core js counts in total execution time)
Is there a page where i can see at least an spected release date? I cant even see commits in master with V8.0.0

**Comment by adamraine** [comment]
The expected release date is today.

**Reply by adamraine** [reply]
Also, a rough outline of our release schedule is mentioned in https://github.com/GoogleChrome/lighthouse/blob/master/docs/releasing.md if you're interested.
